thirdparty/DiskANN/include/utils.h

```diff
@@ -885,7 +885,7 @@ namespace diskann {
     return disk_index_filename + "_centroids.bin";
   }
 
-  inline std::string get_sample_data_filename(const std::string& prefix) {
+  inline std::string get_sample_data_filename(std::string prefix) {
```

Comment:
no, please keep it as `const std::string&` :)

Reply:
updated